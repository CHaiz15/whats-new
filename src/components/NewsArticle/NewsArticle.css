.NewsArticle {
  border: 3px white solid;
  color: white;
  width: 20%;
}

img {
  width: 100%;
}

.site-link {
  background-color: yellow;
}
