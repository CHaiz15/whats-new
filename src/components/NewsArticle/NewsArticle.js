import React from 'react';
import './NewsArticle.css';

const NewsArticle = (props) => {
  return (
    <div className='NewsArticle'>
      <img src={props.img}/>
      <div>
        <h3>{props.headline}</h3>
        <p>{props.description}</p>
      </div>
      <div class='site-link'>
        <a href={props.url}>Click 4 Website</a>
      </div>
    </div>
  );
}


export default NewsArticle;
