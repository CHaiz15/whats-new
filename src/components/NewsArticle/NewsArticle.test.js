import React from 'react'
import { shallow } from 'enzyme'
import NewsArticle from './NewsArticle.js'

describe('NewsArticle', () => {
  it('renders an indication of a NewsArticle', () => {
    const wrapper = shallow(<NewsArticle
      headline='Broncos lose'
      img='bronco.png'
      url='gobroncos.com'
      description='Broncos miss the playoffs'
    />);

    expect(wrapper).toMatchSnapshot();
  })
})
