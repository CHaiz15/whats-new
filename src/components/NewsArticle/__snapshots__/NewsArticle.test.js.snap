// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewsArticle renders an indication of a NewsArticle 1`] = `
<div
  className="NewsArticle"
>
  <img
    src="bronco.png"
  />
  <div>
    <h3>
      Broncos lose
    </h3>
    <p>
      Broncos miss the playoffs
    </p>
  </div>
  <div
    className="site-link"
  >
    <a
      href="gobroncos.com"
    >
      Click 4 Website
    </a>
  </div>
</div>
`;
